include/glow/Quantization/Base/Base.h

```diff
@@ -99,6 +101,13 @@ QuantizationTransform32To8 quantizeScaleOffset32To8(float scale,
 TensorQuantizationParams chooseQuantizationParams(float min, float max,
                                                   Schema schema = Asymmetric);
 
+/// Row-wise quantize the tensor \p input. The param \p input is a 2-D
+/// tensor (i.e. M * N), \p scales and \p offsets are generated by each row of
+/// \p input, \p qData is 2D tensor quantized from \p input using \p scales and
+/// \p offsets for each row.
+void rowWiseQuantization(Tensor &input, Tensor *qData, Tensor *scales,
```

Comment:
Can we do `const Tensor &input` to denote that the function does not change it?

Reply:
I tried this one, but here was an error "note: candidate function not viable: 'this' argument has type 'const glow::Tensor', but method is not marked const
template <class ElemTy> Handle<ElemTy> Tensor::getHandle() {"